zerver/tests/test_auth_backends.py

```diff
@@ -1957,7 +2079,7 @@ def test_github_oauth2_email_no_reply_dot_github_dot_com(self) -> None:
                                                  " emails associated with the account")
 
     def test_github_oauth2_email_not_associated(self) -> None:
-        account_data_dict = dict(email='not-associated@zulip.com', name=self.name)
+        account_data_dict = self.get_account_data_dict(email='not-associated@zulip.com', name=self.name)
```

Comment:
It looks like this batch of 3 little changes to use this existing `get_account_data_dict` helper could have been extracted as a separate preparatory cleanup commit, which I'd recommend you doing for practice.

Reply:
About this, how should I proceed? Do I do another PR with this change? Or I work to make this PR with the two separate commits?